lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -90,30 +75,53 @@ sub list_ajax {
         prefetch => [qw(children parents)],
     )->all;
 
-    # need to use all as the order is too complex for a cursor
-    my $comment_count = $self->prefetch_comment_counts([map { $_->id } @jobs]);
+    my $comment_data = $self->schema->resultset('Comments')->comment_data_for_jobs(\@jobs, {bugdetails => 1});
     my @list;
     for my $job (@jobs) {
         my $job_id = $job->id;
+        my $cd     = $comment_data->{$job_id} // 0;
```

Comment:
What is the `// 0` for here?

Reply:
Oh, that's a leftover from a first version of the code

Reply:
fixed